.github/workflows/e2e-tests.yaml

```diff
@@ -147,4 +147,4 @@ jobs:
           ## limits ssh access and adds the ssh public key for the user which triggered the workflow
           limit-access-to-actor: true
           ## limits ssh access and adds the ssh public keys of the listed GitHub users
-          limit-access-to-users: sandstromviktor,churnikov,hamzaimran08,alfredeen,akochari
+          limit-access-to-users: churnikov,hamzaimran08,alfredeen,akochari,anondo1969
```

Comment:
Damn, good catch. Should add this to the off-boarding process if not already there.